classes/forbidden_shop/ForbiddenShopManager.php

```diff
@@ -11,6 +11,19 @@ public function __construct(System $system, User $player) {
         $this->player = $player;
     }
 
+    const AYAKASHI_FAVOR = 131;
+
+    const FAVOR_EXCHANGE = [
+        127 => 1000, // forbidden scroll
```

Comment:
```suggestion
        LanternEvent::$static_item_ids['forbidden_jutsu_scroll_id'] => 1000, // forbidden scroll
```

Reply:
This might not work, maybe we need to change the lantern event id array to a constant. But I think we should try to keep it as a single source of truth for what ID each item is